### ISSUE TITLE
CONDA_INSTALLER?

### ISSUE THREAD

**Issue by mkandes** [zonca/expanse_singularity#13]
@zonca - I'll submit some pull requests from my end over the weekend. There are simply too many mistakes here that need some fixing up. No worries. I know giving my first tutorial on this process myself may have been a bit confusing too for some to follow along.

**Comment by mkandes** [comment]
@zonca - I've walked through most of the steps here myself with respect to the new sylabs builder section. However, I haven't had a chance yet to review the Singularity documentation on creating encrypted containers and signing them before pushing them to the cloud(s). But I have pushed my modified example from the tutorial to the GHCR. 

```
[xdtr42@login01 miniconda]$ singularity push --docker-username mkandes --docker-password "${GH_PAT}" miniconda_astropy.sif oras://ghcr.io/mkandes/naked-singularity:miniconda-astropy-42.sif
INFO:    Upload complete
```

https://github.com/users/mkandes/packages/container/naked-singularity/41116437?tag=miniconda-astropy-42.sif

Maybe I'll have some time this weekend to draft up some pull requests for my suggested changes. However, there may be more important work for me to do over the weekend. So, my apologies for any delay again on my end.

https://www.crimson-research-institute.com/
